src/components/schemas/report.yaml

```diff
@@ -488,6 +488,47 @@ report_action_with_expenses_ids_to_eject:
           - $ref: './fields.yaml#/id_string'
       example: [ 'txelntgwgwll' ]
 
+report_comments_in:
+  type: object
+  additionalProperties: False
+  required:
+    - id
+    - comment
+  properties:
+    id:
+      type: string
+      description: |
+        id of report, on which need to make comment
```

Comment:
`id of report, on which need to make comment` -->         `id of the report, on which need to add comment`